app/models/event.rb

```diff
@@ -0,0 +1,9 @@
+class Event < ApplicationRecord
+  validates_presence_of :location, :event_date, :description
+  has_many :users_events
+  has_many :attendees, through: :users_events
+  belongs_to :creator, class_name: 'User', foreign_key: 'user_id'
```

Comment:
**A suggestion:** You could have accomplished this by creating a column called `creator_id` and building the association using that name: `belongs_to :creator, class_name: "User"`
That would make your code easier to understand and you wouldn't need to specify the `foreign_key`